ui: fix incorrect display of table replica count

vladlos opened the pull request:
was shown indices instead of replicas.

Resolves: #48599

Release note (ui): Previously the replicas count on the table details page was incorrectly showing number of indices. This has been corrected to show the number of replicas"

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/49185)
<!-- Reviewable:end -->

dhartunian reviewed (changes requested):
> ui: Table details

commit + PR message title should be more descriptive like: "fix incorrect display of table replica count"

> Release note (ui): fix replicas count on table details page

Release note should be more descriptive about what was wrong before: "previously the replicas count on the table details page was incorrectly showing number of indices. This has been corrected to show the number of replicas"

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/49185#-:-M7buPqg3iPrV7NaU8Zd:b-2vjrxv)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @elkmaster)

___
*[pkg/ui/src/views/databases/containers/tableDetails/index.tsx, line 131 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/49185#-M7bxLxN1yXyPCb8tPaj:-M7bxLxO2d-UV4H7-SeP:bwqno1y) ([raw file](https://github.com/cockroachdb/cockroach/blob/53f7f11f03715b9b3740712e46214807b809fe92/pkg/ui/src/views/databases/containers/tableDetails/index.tsx#L131)):*
> ```XML
>                         <Col span={12}>
>                           <div className="summary--card__counting">
>                             <h3 className="summary--card__counting--value">{tableInfo.numReplicas}</h3>
> ```

do we have any other place this number is reported that we should change? 


<!-- Sent from Reviewable.io -->

vladlos committed:
ui: fix incorrect display of table replica count

was shown indices instead of replicas.

Resolves: #48599

Release note (ui): Previously the replicas count on the table details page was incorrectly showing number of indices. This has been corrected to show the number of replicas"

vladlos reviewed (commented):
updated commit message

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/49185#-:-M7c7G2XC7Y6gSt4owtJ:bcy7ii0)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @dhartunian)

___
*[pkg/ui/src/views/databases/containers/tableDetails/index.tsx, line 131 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/49185#-M7bxLxN1yXyPCb8tPaj:-M7c76aHB2WwH7TmzGpB:b-y5zb6) ([raw file](https://github.com/cockroachdb/cockroach/blob/53f7f11f03715b9b3740712e46214807b809fe92/pkg/ui/src/views/databases/containers/tableDetails/index.tsx#L131)):*
<details><summary><i>Previously, dhartunian (David Hartunian) wrote…</i></summary><blockquote>

do we have any other place this number is reported that we should change? 
</blockquote></details>

there is no other places where we show replicas of single table


<!-- Sent from Reviewable.io -->

dhartunian reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/49185#-:-M7cTPki9zRddfyKdP1n:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @elkmaster)

___
*[pkg/ui/src/views/databases/data/tableInfo.tsx, line 40 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/49185#-M7cS3wiDA8m9UYwGAvK:-M7cS3wiDA8m9UYwGAvL:b-3fkf8l) ([raw file](https://github.com/cockroachdb/cockroach/blob/3c207d67c80bd3ae8b37e9c8e5fb32b54df17bf0/pkg/ui/src/views/databases/data/tableInfo.tsx#L40)):*
> ```XML
>       this.createStatement = details && details.create_table_statement;
>       this.grants = details && details.grants;
>       this.numReplicas = details && details.zone_config && details.zone_config.num_replicas;
> ```

@piyush-singh just a sanity check, the number of replicas is being pulled from the zone configuration. does that make sense here?


<!-- Sent from Reviewable.io -->

dhartunian commented:
bors r+
